Registry/RepositoryRegistry.php

```diff
@@ -46,6 +46,27 @@ public function __construct(
     /**
      * {@inheritdoc}
      */
+    public function names()
```

Comment:
Thinking about this again it makes sense - this class could be an iterable (although I don't like the idea), it would not make sense to return an iterator from `all()`.